src/plugins/WebOfTrust/util/BackgroundJobFactory.java

```diff
@@ -0,0 +1,50 @@
+package plugins.WebOfTrust.util;
+
+/**
+ * Factory for {@link BackgroundJob}s which allows all background jobs constructed using an instance
+ * of this class to be terminated with a single method invocation.
+ *
+ * @author bertm
```

Comment:
FYI: I usually tag my files this way so the attribution stays in place even if the file is moved without Git being involved. The main purpose of knowing who wrote it is that questions about its workings can be addressed to the correct person, I usually don't add such a tag unless I (re)write an entire file.